resources/build/utils.inc.sh

```diff
@@ -88,7 +32,9 @@ write_download_info() {
 
   local DATE HASH SIZE DOWNLOAD_INFO STAT_FLAGS
 
-  _verify_platform "${PLATFORM}"
+  if [[ ! "${PLATFORM}" =~ ^(android|ios|linux|mac|web|win)$ ]]; then
```

Comment:
Note: removed 'lmlayer' as a valid platform here. This seems much simpler than the previous function?